src/components/dashboard/enrollments/AllEnrollments.tsx

```diff
@@ -96,7 +96,6 @@ const AllEnrollments = () => {
           rowLinkTo={rowLinkTo}
           columns={columns}
           pagePath='client.enrollments'
-          fetchPolicy='cache-and-network'
```

Comment:
Why is this removed?

Reply:
No reason, probably should be reverted. I went ahead and did that